.github/workflows/ci.yaml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.3"
+          - "2.4"
+          - "2.5"
+          - "2.6"
```

Comment:
I'm not sure about this project's support policy though, trying to keep the current test targets.